bin/rad-project

```diff
@@ -118,9 +118,9 @@
           "1. New peer-to-peer repository (git-ipfs **EXPERIMENTAL**)"
           "2. Add your own remote (e.g.: github / gitlab / ...)"]
          (match current-remote
-               "" []
+               "" [""]
                'url ["3. Remote origin of existing current git repository:"
-                     (string-append "   " url)])
+                     (string-append "   " url "\n")])
          ])))
     (match (prompt! question)
```

Comment:
Maybe it would be clearer to just replace `question` here with `(string-append question "\n")`.